### DISCUSSION TITLE
reassign variable to another type

### DISCUSSION THREAD

**Discussion by Kostia06** [vlang/v#15858] [General]
I have this issue where I need my variable either to be an interger or an string.
'''
condition = true
mut var
if condition{var = "pop"}
else{var = 1}
'''
Is there a way to do that?

**Comment by JalonSolov** [comment]
No.  V is statically typed.  Once the type of a variable is set, you cannot change the type.

**Reply by JalonSolov** [reply]
You _could_ use a sum type, instead of a regular variable.

https://github.com/vlang/v/blob/master/doc/docs.md#sum-types
